uaclient/entitlements/base.py

```diff
@@ -153,6 +166,78 @@ def can_enable(self, silent: bool = False) -> bool:
             if not silent:
                 print(details)
             return False
+
+        if self.incompatible_services:
+            return self.handle_incompatible_services()
+
+        return True
+
+    def handle_incompatible_services(self):
+        """
+        Handle to proceed when a incompatible service is found during enable
+
+        When enabling a service, we may find that there is an incompatible
+        service already enable. In that situation, we can ask the user
+        if the incompatible service should be disabled before proceeding.
+        There are also different ways to configure that behavior:
+
+        We can disable removing incompatible service during enable by
+        adding the following lines into uaclient.conf:
+
+        features:
+          block_disable_on_enable: true
+
+        We can also use the --allow-disable flag during enable to
+        automatically disable any incompatible service during
+        enable.
+        """
+        from uaclient.entitlements import ENTITLEMENT_CLASS_BY_NAME
+
+        cfg_block_disable_on_enable = util.is_config_value_true(
+            config=self.cfg.cfg,
+            path_to_value="features.block_disable_on_enable",
+        )
```

Comment:
I don't think we need to surface yet another config option/feature override for this do we? Shouldn't our default behavior be to block disable of livepatch on when enabling FIPS and allow a user to either interact with the prompt with "Y"  or provide a `--force-yes` option.

Reply:
The problem is related to behave tests. I couldn't find a direct way to send a user input through behave. Therefore, this flag is way to test the scenario where a user doesn't want to disable livepatch during the install

Reply:
+1 I couldn't find a good way to do this through BDD on initial glance. Let's land this iteration and take a tech-debt item #1327 to better provide input "answers" via BDD that doesn't also surface more "public-facing" config parameters in the features: section.